wallet/src/wallet.rs

```diff
@@ -23,6 +23,11 @@ use std::path::Path;
 
 pub type CapeWalletError = WalletError<CapeLedger>;
 
+pub fn default_erc20_code() -> Erc20Code {
+    let zeros: [u8; 20] = [0; 20];
+    Erc20Code(EthereumAddr(zeros))
+}
```

Comment:
Where is this used?

Reply:
Oops, this was left over from before I merged in the fix for `is_wrapped_asset` to this branch.